spread.yaml

```diff
@@ -82,6 +82,8 @@ environment:
     SNAPD_NO_MEMORY_LIMIT: '$(HOST: echo "${SPREAD_SNAPD_NO_MEMORY_LIMIT:-}")'
 
     SNAPD_PUBLISHED_VERSION: '$(HOST: echo "$SPREAD_SNAPD_PUBLISHED_VERSION")'
```

Comment:
How is SNAPD_PUBLISHED_VERSION different from SNAPD_DEB_FROM_REPO?

Reply:
Given that you know, can you add a comment to both so that it is clear when to use which.

Reply:
perhaps we could also remove SNAPD_PUBLISHED_VERSION
I think it was not used since very long time